<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Yoklama Sistemi</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                               stop:0 #f5f7fa, stop:1 #c3cfe2);
    color: #2c3e50;
    font-family: 'Segoe UI', Arial, sans-serif;
    font-size: 10pt;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">QStackedWidget {
    background: rgba(255, 255, 255, 0.9);
    border: 2px solid #3498db;
    border-radius: 10px;
}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="placeholderWidget">
       <property name="styleSheet">
        <string notr="true">QWidget {
    background: rgba(255, 255, 255, 0.9);
    border: 2px solid #3498db;
    border-radius: 10px;
}</string>
       </property>
       <layout class="QVBoxLayout" name="placeholderLayout">
        <item>
         <widget class="QLabel" name="placeholderLabel">
          <property name="styleSheet">
           <string notr="true">QLabel {
    font-size: 18pt;
    font-weight: bold;
    color: #2c3e50;
    padding: 20px;
}</string>
          </property>
          <property name="text">
           <string>Yoklama Sistemi - Ana Pencere</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="infoLabel">
          <property name="styleSheet">
           <string notr="true">QLabel {
    font-size: 10pt;
    color: #7f8c8d;
    padding: 10px;
}</string>
          </property>
          <property name="text">
           <string>Bu pencere Qt Designer önizlemesi için kullanılır.
Gerçek uygulamada Login, Admin ve Teacher widget'ları burada görüntülenir.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                               stop:0 #34495e, stop:1 #2c3e50);
    color: white;
    font-weight: bold;
}</string>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">QStatusBar {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                               stop:0 #34495e, stop:1 #2c3e50);
    color: white;
    font-weight: bold;
    padding: 5px;
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
